hashes/sdbm.py

```diff
@@ -26,12 +26,12 @@ def sdbm(plain_text: str) -> str:
     them.
 
     >>> sdbm('Algorithms')
-    1462174910723540325254304520539387479031000036
+    '1462174910723540325254304520539387479031000036'
 
     >>> sdbm('scramble bits')
-    730247649148944819640658295400555317318720608290373040936089
+    '730247649148944819640658295400555317318720608290373040936089'
     """
     hash = 0
     for plain_chr in plain_text:
         hash = ord(plain_chr) + (hash << 6) + (hash << 16) - hash
-    return hash
+    return str(hash)
```

Comment:
Given that the sdbm code sited at https://en.wikipedia.org/wiki/List_of_hash_functions returns an int, I would prefer that we did so as well.